spec/controllers/general_controller_spec.rb

```diff
@@ -275,7 +275,7 @@
   # rebuild xapian index after fixtures loaded
   before(:each) do
     load_raw_emails_data
-    get_fixtures_xapian_index
+    update_xapian_index
```

Comment:
I think we should note in the commit message that the _reason_ we're doing this is that (AIUI) we now create an initial index, and then after that we only need to reindex based on the new test data added. We're also letting rspec handle the fixture loading.

Reply:
Update commit message